### PR TITLE
Add test with utf8 values

### PR DESCRIPTION
**Pull request by macintux**

### CODE REVIEW

File: tests/ts_unicode.erl [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    io:format("Got ~b rows back again\n", [length(Rows)]),
+    ?assertEqual(length(Rows), 10 - 1 - 1),
+
+    pass.
+
+
+%% @ignore
+-spec build_cluster(non_neg_integer()) -> [node()].
+build_cluster(Size) ->
+    build_cluster(Size, [{yokozuna, [{enabled, true}]}]).
```
Comment (javajolt) [inline tests/ts_unicode.erl]: Why do you have YZ enabled?

Comment (macintux) [inline tests/ts_unicode.erl]: Legacy from ts_basic, whence this script.

Comment (javajolt) [inline tests/ts_unicode.erl]: I'll +1 it for now.  Test passes.

**Comment by javajolt** [comment]
:+1:

**Pull request merged**